### PR TITLE
Update WSL install guides

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/computer/wsl-setup.md [hunk 1]
```diff
@@ -25,20 +25,40 @@
...
+WSL 只能在 Windows 10 的较高版本或 Windows 11 上安装，因而，在安装 WSL 前，需要
+先检查当前 Windows 系统的版本号是否满足 WSL 的要求。
+
+按下 {kbd}`win` + {kbd}`R` 键，在打开的运行对话框中键入 `winver`，然后选择“确定”，
+则会弹出“关于 Windows”对话框，会看到类似“版本 2004 （OS 内部版本 19041 ）”的字样。
+其中，2004 是 Windows 版本号，19041 是系统内部版本号。
+
+### 安装 WSL
+
+若你使用的是 Windows 10 版本 版本 2004 及更高版本（内部版本 19041 及更高版本）
```
Comment (seisman) [inline source/computer/wsl-setup.md:39]:
```suggestion
若你使用的是 Windows 10 版本 2004 及更高版本（内部版本 19041 及更高版本）
```

File: source/computer/wsl-setup.md [hunk 2]
```diff
@@ -25,20 +25,40 @@
...
+其中，2004 是 Windows 版本号，19041 是系统内部版本号。
+
+### 安装 WSL
+
+若你使用的是 Windows 10 版本 2004 及更高版本（内部版本 19041 及更高版本）
+或 Windows 11，则可以通过如下方式安装 WSL。
+
+1.	在 Windows 系统的搜索框中直接输入命令 `CMD`
+2.	在搜索结果中的“命令提示符”上单击右键，选择管理员身份运行
+3.	在打开的 CMD 窗口中，输入如下命令：
```
Comment (core-man) [inline source/computer/wsl-setup.md:44]:
```suggestion
1.  在 Windows 系统的搜索框中直接输入命令 `CMD`
2.  在搜索结果中的“命令提示符”上单击右键，选择管理员身份运行
3.  在打开的 CMD 窗口中，输入如下命令：
```

File: source/computer/wsl-setup.md [hunk 3]
```diff
@@ -25,20 +25,40 @@
...
+	```
+	wsl --install
+	```
+	此命令将启动 WSL 并默认安装 Ubuntu 22.04 LTS
```
Comment (core-man) [inline source/computer/wsl-setup.md:48]: Tap or black space?

Comment (seisman) [reply source/computer/wsl-setup.md]: Now all use whitespaces.

**Review by core-man — APPROVED** [review]

**Pull request merged**